### DISCUSSION TITLE
Multi in Uni combine

### DISCUSSION THREAD

**Discussion by uPagge** [quarkusio/quarkus#23170] [Q&A]
Is it possible to cram Multi into Uni.combine().all() ?

When I try to somehow convert Multi to Uni and pass it to Uni.combine, I get this

Connection was closed: io.vertx.core.VertxException: Connection was closed

**Comment by kenyee** [comment]
A Uni only emits once but you're feeding something that emits multiple events to it.
If you take one from the Multi by doing toUni(), it should work...

**Reply by uPagge** [reply]
and what will I get by doing toUni? Random element from Multi? I need a collection of elements in order to do something with it. Uni<List<?>>

But when I do it on the receiving service side, converting Multi to Uni<List>, I get this error. And if I do it on the side of the giving service, that is, I get Multi from the database, then I convert it to Uni<List> and give it to another service, then everything works

**Comment by kenyee** [comment]
you'll get the first element by doing toUni().
Multi means multiple emissions of an object.

Can you show some code?  Not sure what you're trying to do....
